### DISCUSSION TITLE
ERROR: Model provided has model identifier 'ion ', should be 'TFL3'

### DISCUSSION THREAD

**Discussion by wwbnjsace** [usefulsensors/openai-whisper#10] [Q&A]
when i run ,there is an error 

./minimal ../models/whisper.tflite ../samples/jfk.wav 

n_vocab:50257

mel.n_len3000

mel.n_mel:80
ERROR: Model provided has model identifier 'ion ', should be 'TFL3'

Error at /home/smart-ns/ns/tflite-whisper/openai-whisper-main/tensorflow_src/tensorflow/lite/examples/minimal/minimal.cc:179

**Comment by nyadla-sys** [comment]
@wwbnjsace Would you kindly supply further information, such as the hardware you used and the OS?

**Reply by wwbnjsace** [reply]
> @wwbnjsace Would you kindly supply further information, such as the hardware you used and the OS?

i test in a ubuntu server ,8 cpu ,30 mem, 
i do not use the git lfs , i use git lfs try again, in the 'cmake ../tensorflow_src/tensorflow/lite/examples/minimal' stage ,there is anther error  .
CMake Error at /data/home/nieshuai/tflite-whisper/openai-whisper/minimal_build/cpuinfo/CMakeLists.txt:262 (ADD_SUBDIRECTORY):
  ADD_SUBDIRECTORY not given a binary directory but the given source
  directory
  "/data/home/nieshuai/tflite-whisper/openai-whisper/minimal_build/clog-source"
  is not a subdirectory of
  "/data/home/nieshuai/tflite-whisper/openai-whisper/minimal_build/cpuinfo".
  When specifying an out-of-tree source a binary directory must be explicitly
  specified.


CMake Error at /data/home/nieshuai/tflite-whisper/openai-whisper/minimal_build/cpuinfo/CMakeLists.txt:265 (SET_PROPERTY):
  SET_PROPERTY could not find TARGET clog.  Perhaps it has not yet been
  created.


CMake Error at /data/home/nieshuai/tflite-whisper/openai-whisper/minimal_build/xnnpack/CMakeLists.txt:7454 (ADD_SUBDIRECTORY):
  ADD_SUBDIRECTORY given source
  "/data/home/nieshuai/tflite-whisper/openai-whisper/minimal_build/clog-source/deps/clog"
  which is not an existing directory.


CMake Error at /data/home/nieshuai/tflite-whisper/openai-whisper/minimal_build/xnnpack/CMakeLists.txt:7458 (SET_PROPERTY):
  SET_PROPERTY could not find TARGET clog.  Perhaps it has not yet been
  created.


CMake Error at /data/home/nieshuai/tflite-whisper/openai-whisper/minimal_build/xnnpack/CMakeLists.txt:7558 (ADD_SUBDIRECTORY):
  The source directory

    /data/home/nieshuai/tflite-whisper/openai-whisper/minimal_build/FP16-source

  does not contain a CMakeLists.txt file.


-- Configuring incomplete, errors occurred!
See also "/data/home/nieshuai/tflite-whisper/openai-whisper/minimal_build/CMakeFiles/CMakeOutput.log".
See also "/data/home/nieshuai/tflite-whisper/openai-whisper/minimal_build/CMakeFiles/CMakeError.log".

**Comment by nyadla-sys** [comment]
Please run git lfs install before git clone to repository
